lib/wallaby/experimental/chrome.ex

```diff
@@ -183,7 +183,17 @@ defmodule Wallaby.Experimental.Chrome do
   @doc false
   def click(element), do: delegate(:click, element)
   @doc false
-  def hover(element), do: delegate(:hover, element)
+  def click(parent, button), do: delegate(:click, parent, [button])
+  @doc false
+  def double_click(parent), do: delegate(:double_click, parent)
+  @doc false
+  def button_down(parent, button), do: delegate(:button_down, parent, [button])
+  @doc false
+  def button_up(parent, button), do: delegate(:button_up, parent, [button])
+  @doc false
+  def hover(element), do: delegate(:move_mouse_to, element, [element])
+  @doc false
+  def move_mouse_by(parent, x_offset, y_offset), do: delegate(:move_mouse_to, parent, [nil, x_offset, y_offset])
```

Comment:
I thought about it, but finally I decided to name this function `move_mouse_by`, because it only moves mouse _by_ an offset from the current position; `move_mouse_to` suggests moving to the given point or to the given element, I think. On the other hand, moving _by_ an offset from the current position is also moving _to_ some position, but still I think that `move_mouse_by` is a bit more intuitive name for this function.

Reply:
Ah, I see. Hover is really `move_mouse_to` because you pass the element that you want to move the cursor over.

👌

Reply:
Exactly. :)